common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1417,6 +1417,13 @@ def save_problem(self, data):
             }
 
         self.lcp.student_answers = answers
+        answer_keys = [key for key in answers]
+        for answer_key in answer_keys:
+            # Reset the correctness property to default(None) state when attempted wrong
+            if self.lcp.correct_map.get_property(answer_key, 'correctness', None) == 'incorrect':
```

Comment:
hmmm, do we want to be reseting "correctness" to `None`? The student still got the problem incorrect. Will this affect grading? Statistics that installations will want to run on CSM? Should we just add some calculation or field `should_show_check_mark` that checks if we've saved after we've last pressed "check"?